nox/virtualenv.py

```diff
@@ -220,8 +245,13 @@ def _resolved_interpreter(self):
         if self._resolved is not None:
             return self._resolved
 
+        currently_in_virtual_environment = sys.prefix != sys.base_prefix
+
         if self.interpreter is None:
-            self._resolved = sys.executable
+            if currently_in_virtual_environment:
+                self._resolved = resolve_real_python_outside_venv(self.interpreter)
```

Comment:
As it stands, this has no hope of working on Windows. At this point, we're calling `resolve_real_python_outside_venv(None)` (side note, it might be better to explicitly use `None`, rather than have the reader need to track back to the if statement to determine that `self.interpreter` is always `None` at this point). There's no check for `_WINDOWS` here (unlike the call below) so the fact that `resolve_real_python_outside_venv` doesn't (yet) support Windows will cause immediate breakage.

Reply:
Agreed